src/js/components/channels/transfer-request-channel-content.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { hideChannelPage } from '../../services/app';
+import { LoadingComponent } from '../../components/loading';
+import { resetTransferRequestCode, unsetError } from '../../actions/integrations-actions';
+import { fetchTransferRequestCode } from '../../services/integrations';
+
+export class TransferRequestChannelContentComponent extends Component {
+
+    static propTypes = {
+        channelState: PropTypes.object.isRequired,
+        transferError: PropTypes.string
+    };
+
+    onLink = () => {
+        const {dispatch, type} = this.props;
+        dispatch(resetTransferRequestCode(type));
```

Comment:
if those actions aren't async (using thunks), you can batch them using `batchActions`. It will optimize rendering.